accelerator/core/mam_core.c

```diff
@@ -776,3 +776,29 @@ status_t ta_recv_mam_message(const iota_config_t *const iconf, const iota_client
   mam_encrypt_key_free(&mam_key);
   return ret;
 }
+
+status_t ta_register_mam_channel(const ta_cache_t *const cache, const ta_register_mam_channel_req_t *const req,
+                                 char *uuid) {
+  if (!req || !uuid) {
+    ta_log_error("%s\n", ta_error_to_string(SC_NULL));
+    return SC_NULL;
+  }
+  status_t ret = SC_OK;
+
+  uuid_t binuuid;
```

Comment:
`binuuid` is misleading. What is your intention?

Reply:
Let me rename it `bin_uuid` which means binary uuid